Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -174,15 +167,15 @@ class SignalAttachment: NSObject {
 
     // Returns the MIME type for this attachment or nil if no MIME type
     // can be identified.
-    var mimeType: String? {
+    var mimeType: String {
         if dataUTI == SignalAttachment.kOversizeTextAttachmentUTI {
             return OWSMimeTypeOversizeTextMessage
         }
         if dataUTI == SignalAttachment.kUnknownTestAttachmentUTI {
             return OWSMimeTypeUnknownForTests
         }
         guard let mimeType = UTTypeCopyPreferredTagWithClass(dataUTI as CFString, kUTTagClassMIMEType) else {
-            return nil
+            return OWSMimeTypeApplicationOctetStream
```

Comment:
We want to send all attachments. We need *some* mimetype for any attachment we send, so use a generic mimetype when we can't ascertain one from the UTI or extension.